packages/core/aqua-to-js/src/generate/__test__/generate.spec.ts

```diff
@@ -14,46 +14,55 @@
  * limitations under the License.
  */
 
-import url from "url";
+import { fileURLToPath } from "url";
 
 import { compileFromPath } from "@fluencelabs/aqua-api";
-import { describe, expect, it } from "vitest";
+import { beforeAll, describe, expect, it } from "vitest";
 
 import { getPackageJsonContent, PackageJson } from "../../utils.js";
 import { generateTypes, generateSources } from "../index.js";
+import { CompilationResult } from "../interfaces.js";
+
+let res: Omit<CompilationResult, "funcCall">;
+let pkg: PackageJson;
 
 describe("Aqua to js/ts compiler", () => {
-  it("compiles smoke tests successfully", async () => {
-    const res = await compileFromPath({
-      filePath: url.fileURLToPath(
+  beforeAll(async () => {
+    res = await compileFromPath({
+      filePath: fileURLToPath(
         new URL("./sources/smoke_test.aqua", import.meta.url),
```

Comment:
can you please write some comments on what files this produces and how they will be used?

Reply:
These files written in __snapshot__folder as you can find below. This is documented behavior in vitest.